<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo Organizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>850</width>
      <height>580</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>650</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame{
border: 1.5px solid #8ba5c6;
background-color: rgb(247, 247, 247);
border-radius: 10px;
}







</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>591</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Roboto&quot;;
font-style:italic;
border-color: rgb(247, 247, 247);
background-color: rgb(247, 247, 247);</string>
      </property>
      <property name="text">
       <string>Before uploading to dashboard please check if every qr code and
folder was processed correctly.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>381</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Roboto&quot;;
border-color: rgb(247, 247, 247);
background-color: rgb(247, 247, 247);</string>
         </property>
         <property name="text">
          <string>EVENT FOLDER   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="eLabel">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Roboto&quot;;
border-color: rgb(247, 247, 247);
font-weight:bold;
background-color: rgb(247, 247, 247);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(247, 247, 247);
font: 12pt &quot;Roboto&quot;;
background-color: rgb(247, 247, 247);</string>
         </property>
         <property name="text">
          <string>Is created in you directory  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewButton">
         <property name="styleSheet">
          <string notr="true">
border:none;
color: rgb(135, 169, 202);
text-decoration: underline;
font: 10pt &quot;Askhar&quot;;
font-weight: bold;
background-color: rgb(247, 247, 247);
</string>
         </property>
         <property name="text">
          <string>View Folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(247, 247, 247);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>371</width>
        <height>85</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Pacifico</family>
           <pointsize>28</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(247, 247, 247);
color: rgb(135, 169, 202);
background-color: rgb(247, 247, 247);</string>
         </property>
         <property name="text">
          <string>Thank You</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="dashboardButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>470</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 195, 79);
border-bottom-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
border-left-color: rgb(247, 247, 247);
border-left-color: rgb(0, 0, 0);
border: 1px solid #000000;
border-radius: 5;
font: 8pt &quot;Roboto&quot;;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>open my photographer dashboard online</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>470</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #e9acbc;
border-bottom-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
border-left-color: rgb(247, 247, 247);
border-left-color: rgb(0, 0, 0);
border: 1px solid #000000;
border-radius: 5;
font: 8pt &quot;Roboto&quot;;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Upload next event</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>-100</y>
       <width>461</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/resources/dino-new.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
